case-study-orchestral/index.html

```diff
@@ -0,0 +1,2567 @@
+    <![endif]-->
+    <!--[if IE 8]>
+      <link
+        rel="stylesheet"
+        type="text/css"
+        href="/wp/wp-content/themes/rise/css/ie8.css"
+      />
+    <![endif]-->
+    <!--[if IE 7]>
+      <link
+        rel="stylesheet"
+        type="text/css"
+        href="/wp/wp-content/themes/rise/css/ie7.css"
+      />
+    <![endif]-->
+
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- WP_HEAD() START -->
+    <script type="text/javascript">
+      function apbct_attach_event_handler__backend(elem, event, callback) {
+        if (typeof window.addEventListener === "function")
+          elem.addEventListener(event, callback);
+        else elem.attachEvent(event, callback);
+      }
+      apbct_attach_event_handler__backend(window, "load", function () {
+        if (typeof apbctLocalStorage === "object")
+          apbctLocalStorage.set("ct_checkjs", "232650532", false);
+        else console.log("APBCT ERROR: apbct-public--functions is not loaded.");
+      });
+    </script>
+    <!-- Search Engine Optimization by Rank Math - https://s.rankmath.com/home -->
+    <title>Case Study - Orchestral</title>
+    <meta
+      name="robots"
+      content="follow, index, max-snippet:-1, max-video-preview:-1, max-image-preview:large"
+    />
+    <link
+      rel="canonical"
+      href="https://stackstorm.com/case-study-orchestral/"
+    />
+    <meta property="og:locale" content="en_US" />
+    <meta property="og:type" content="article" />
+    <meta
+      property="og:title"
+      content="Case Study - Orchestral.ai"
+    />
+    <meta
+      property="og:url"
+      content="https://stackstorm.com/case-study-orchestral/"
+    />
+    <meta property="og:site_name" content="Orchestral.ai" />
+    <meta property="article:author" content="admin@orchestral.ai" />
+    <meta property="og:updated_time" content="2022-07-31T17:37:14-07:00" />
+    <meta
+      property="og:image"
+      content="https://stackstorm.com/case-study-orchestral/images/Orchestral.ai_full_logo.png"
+    />
+    <meta
+      property="og:image:secure_url"
+      content="https://stackstorm.com/case-study-orchestral/images/Orchestral.ai_full_logo.png"
+    />
+    <meta property="og:image:width" content="1200" />
+    <meta property="og:image:height" content="630" />
+    <meta property="og:image:alt" content="Case Study - Orchestral" />
+    <meta property="og:image:type" content="image/png" />
+    <meta
+      property="article:published_time"
+      content="2022-07-25T06:59:16-07:00"
+    />
+    <meta
+      property="article:modified_time"
+      content="2022-07-31T17:37:14-07:00"
+    />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta
+      name="twitter:title"
+      content="Case Study - Orchestral.ai"
+    />
+    <meta
+      name="twitter:image"
+      content="https://stackstorm.com/case-study-orchestral/images/Orchestral.ai_full_logo.png"
```

Comment:
For all the URLs or links that include stackstorm.com, it's better to use the absolute dir/path like this:
```suggestion
      content="/case-study-orchestral/images/Orchestral.ai_full_logo.png"
```
This will ensure the structure works when someone tries to develop it locally.